aces/imaging/TP-ACA-feather.py

```diff
@@ -0,0 +1,200 @@
+# This script is to feather 7m and TP cubes.
+
+
+# A template CMZ cube for mosaic
+#templateIM='/almalustre/home/pyhsieh/ESO/spw24/template/CS2-1-temlate-whole-band.image'   # whole band
+#templateIM='/almalustre/home/pyhsieh/ESO/spw24/template/CS2-1-temlate-narrow-band.image'  # only 0~1500 channel
+templateIM='CS2-1-temlate-whole-band.image'
+
+import glob
+linename=glob.glob('*.image.pbcor')
+linewt=glob.glob('*.weight')
+
+
+fieldnum=38
+
+# update rest frequency of 12m cubes (line and weight data) with CS(2-1) line
+  
+for i in range(len(linename)):
+  imhead( imagename=linename[i],
+    mode='put',
+    hdkey='restfreq',hdvalue='97.980953GHz',verbose=False)
+
+for i in range(len(linewt)):
+  imhead( imagename=linewt[i],
+    mode='put',
+    hdkey='restfreq',hdvalue='97.980953GHz',verbose=False)
+
+
+
+# reading spectral line and weight cubes of 7m and TP from dict.txt
+# in dict-aca.txt, adding the pairs of 7m and TP mous id
+
+daca={}
+dtp={}
+with open("dict-aca.txt") as f:
```

Comment:
could you add this file too?

Reply:
It's added in #342

Reply:
Thanks for the comments. I have edited the script accordingly (I think I dit it). Which bottom should I click to submit this change request so you could merge it?

linename=glob.glob('*.image.pbcor')
linename=glob.glob('*spw33*.image.pbcor')